src/lib/background/export/index.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+
+const {collect} = require("../stats/collector");
+const userStatus = require("../domains/status/user");
+const {getExtensionSettings} = require("./settings");
+const flattrManager = require("../flattrManager");
+
+function exportExtensionData()
+{
+  return Promise.all([
+    collect({skipSubstitution: true}),
+    flattrManager.query({flattrs: true, start: 0}),
+    userStatus.getAll(),
+    getExtensionSettings()
+  ])
+  .then(([
+    browsingEvents,
+    flattrsPending,
+    userDomainsSettings,
+    userExtensionSettings
+  ]) =>
+  {
+    return {
+      browsingEvents,
+      flattrsPending,
+      userDomainsSettings,
+      userExtensionSettings
```

Comment:
Just for reference, we could also include the contents of the other databases. Although I'm fine with not including it for now to keep this review simple since they're cleared daily and can be inferred from either `browsingEvents` or the browser's history data.